src/App.vue

```diff
@@ -3,8 +3,10 @@
     <div class="container mb-4 mt-4">
       <div class="mb-4">
         <h1 class="logo">
-          <img alt="Logo" src="./assets/logo.png" width="50">
-          Open Drinks
+          <router-link to="home">
+            <img alt="Logo" src="./assets/logo.png" width="50" />
+            Open Drinks
+          </router-link>
```

Comment:
This also makes the text blue and underline since it's a link. Can you fix so the color remains the same, and there's no underline when hovered?

Reply:
i have fixed it. Can u check again

Reply:
On other pages, it seems to still remain blue (explore/search).

Reply:
> On other pages, it seems to still remain blue (explore/search).

oh sorry. i've just fixed it .